docs/_includes/common/floating-label.html

```diff
@@ -245,6 +245,73 @@ <h3>Floating Label with long text</h3>
 </span>
     {% endhighlight %}
 
+    <h3>Floating Label with select</h3>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="select floating-label">
+                <label class="floating-label__label">Select Option</label>
+                <select aria-label="Select demo" name="options">
+                    <option value=""></option>
+                    <option value="item1">Pick Option 1</option>
+                    <option value="item2">Pick Option 2</option>
+                    <option value="item3">Pick Option 3</option>
+                </select>
+                <svg class="icon icon--dropdown" focusable="false" height="8" width="8" aria-hidden="true">
+                    {% include common/symbol.html name="dropdown" %}
+                </svg>
+            </span>
+        </div>
+    </div>
+
+    {% highlight html %}
+<span class="select floating-label">
+    <label class="floating-label__label">Select Option</label>
+    <select name="options">
+        <option value=""></option>
+        <option value="item1">Pick Option 1</option>
+        <option value="item2">Pick Option 2</option>
+        <option value="item3">Pick Option 3</option>
+    </select>
+    <svg class="icon icon--dropdown" focusable="false" height="8" width="8" aria-hidden="true">
+        <use xlink:href="#icon-dropdown"></use>
+    </svg>
+</span>
+    {% endhighlight %}
+
+    <h3>Floating Label with large select</h3>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="select select--large floating-label floating-label--large">
+                <label class="floating-label__label floating-label__label"">Select Option</label>
+                <select aria-label="Select demo" name="options">
+                    <option value=""></option>
+                    <option value="item1">Pick Option 1</option>
+                    <option value="item2">Pick Option 2</option>
+                    <option value="item3">Pick Option 3</option>
+                </select>
+                <svg class="icon icon--dropdown" focusable="false" height="8" width="8" aria-hidden="true">
+                    {% include common/symbol.html name="dropdown" %}
+                </svg>
+            </span>
+        </div>
+    </div>
+
+    {% highlight html %}
+<span class="select select--large floating-label floating-label--large">
```

Comment:
For consistency, can we have the floating label element be a wrapper around the select? Like we do for textboxes. This will also help avoid chained selectors.

Reply:
Done